arroyo-rpc/proto/api.proto

```diff
@@ -409,3 +410,33 @@ message OperatorCheckpointDetail {
   bool has_state = 4;
   map<uint32, TaskCheckpointDetail> tasks = 5;
 }
+
+// Arrow
+message ArrowProgram {
+  repeated ArrowNode nodes = 1;
+  repeated ArrowEdge edges = 2;
+}
+
+message ArroyoSchema {
+  // json-encoded
+  string arrow_schema = 1;
+  uint32 timestamp_col = 2;
+  repeated uint32 key_cols = 3;
+}
+
+message ArrowNode {
+  int32 node_index = 1;
+  string node_id = 2;
+  uint32 parallelism = 3;
+  string description = 4;
+  string operator_name = 5;
+  bytes operator_config = 6;
```

Comment:
Does it make senes for this to be optional? I guess the operator_name could decide to just not look at it.

Reply:
In the current structure, everything needs a config because there's a unified way to construct operators. What kind of operator are you thinking of that doesn't have any config?

Reply:
I was doing without a config for e.g. the GrpcSink, but having a config with an empty table and connection also works.

Reply:
All connectors have the same config (ConnectorOp) like today